lib/scheduler.registry.ts

```diff
@@ -8,6 +8,19 @@ export class SchedulerRegistry {
   private readonly timeouts = new Map<string, any>();
   private readonly intervals = new Map<string, any>();
 
+  doesExists(type: 'cron' | 'timeout' | 'interval', name: string) {
+    switch (type) {
+      case 'cron':
```

Comment:
Hi :)
What about using enum for `type`? Maybe it will be easier to support it.
Cause if someone will add new type, or rename existing, it'll be additional work to change in all files instead of one string in enum?

Reply:
I think here is unnecessary to use enum. Since the type of the task/job is already limited. There may only have interval, timeout, and cronjob.